### ISSUE TITLE
Project Update 02/28

### ISSUE THREAD

**Issue by danakaufhold** [enb34/alice-in-translation#11]
It was a productive week for us here at Alice in Translation! Our [website](alice-jpn.obdurodon.org) is coming along swimmingly, and we hope to have at least blank pages up for all of the pages we'll eventually fill with content. I'm going to be looking into using SSI so that I can more easily include our menu bar on each page instead of copy-pasting the html on every new document I make. 

For our midterm push, we'll be focusing on finishing our markup. Emily has been hard at work, and we're discovering new intricacies in the texts every day. We plan on adding metadata to our documents that explain our tagging, so stay tuned for that. In the meantime, keep checking out our website and let us know any thoughts you may have!

**Comment by JosephDRogers23** [comment]
Can you guys talk about what inspired the design for your website? Is it just Alice in Wonderland, or did the font/clouds background come from something else as well? Is it the overall feeling of whimsy?

**Comment by danakaufhold** [comment]
@JosephDRogers23 Yes, it was mostly just based off the themes of Alice! I knew I wanted to include a web font, so I found something that was fun but not obnoxious for titles and such and then just used a simple sans-serif for the rest of the text. The clouds came about because we had both agreed that we liked the background on the Game of Thrones project, and I wanted to find an image that would tie back to the story at least somehow without being obtrusive or drawing the eye too much.

**Comment by brucknerp** [comment]
Sounds like you guys are making great progress!

I was just looking at both source texts again, and I thought it was cool that the gendered language we're looking at in our project shows up in Alice's dialogue in the Japanese version (sentence-final particles make her speech more feminine than is conveyed in the English). 

Japanese translations usually input some sort of gendered language trait for characters, usually more for females IIRC. I remember watching a scene of Japanese-dubbed Harry Potter one, and Hermione's voice actor and speech lines were way more feminine, giving her an entirely different character, which was interesting and sad :( 

Anyway, can't wait to see more!

**Comment by Idi0teque** [comment]
Really great stuff so far! How do you intend to handle metadata? Like, aside from title and author, will you be listing dates both published and translated? Otherwise, I really don't know what more there is to add to the site; it looks practically finished, and incredible! If you don't mind me asking, was your Individual Character Conclusions page automatically output then manually edited or created entirely through text?

**Comment by danakaufhold** [comment]
@brucknerp That's so interesting! I hope Emily gets to take a look at your comment because it's fascinating. 

@Idi0teque I'm not sure what you're referring to by the Individual Character Conclusions? The only pages on our site that are active right now are the Home page and the page that will contain the annotated texts.

**Comment by djbpitt** [comment]
@enb34 @danakaufhold Terrific use of flexbox! Just a few small housekeeping details:

1. Your GitHub repo doesn’t seem to be in sync with what you have on Obdurodon. You’ll want to do all of the web site development in the repo, and keep it synchronized with what you’re displaying on line.
1. You have both an `obdurodon` and a `web_codes` subdirectory in your repo. I set up the first one with a placeholder index file, and you started working in the second, so you should probably get rid of the first entirely now, since you don’t need it anymore. 
1. You have a small but important error in your `<link>` element in the index and texts pages, which you can find by validating them at <https://validator.w3.org/>. The error is that the pipe, or vertical bar (`|`) isn’t allowed where you use it. This is a well-known issue (it’s sloppiness on Google’s part, and not yours!), and you can fix it by following the explanation at <https://stackoverflow.com/questions/22466913/google-fonts-url-break-html5-validation-on-w3-org> (post a follow-up here if anything about that explanation is unclear). 
1. Now that you have two pages, this is a good time to switch to using a server side include (SSI) for the text that you include on all pages, like your menu. You can read about this at <http://dh.obdurodon.org/ssi.xhtml>. Joe already implemented an SSI on <http://magic.obdurodon.org>, so if anything is unclear, you might want to look into the Magic Realism repo for an example.

Your site is looking great, and we admire the steady progress. And did we mention the terrific use of flexbox?
